s3-supported-api.md

```diff
@@ -0,0 +1,119 @@
+12. Get Bucket tagging  
+    `aws s3api get-bucket-tagging --bucket <your_bucket>`  
+13. Delete bucket tagging  
+    `aws s3api delete-bucket-tagging --bucket <your_bucket>`  
+14. Get bucket location  
+    `aws s3api get-bucket-location --bucket <your_bucket>`  
+
+
+## Supported operations/api on Object
+
+1.  Put Object  
+    `aws s3api put-object --bucket <your_bucket> --key <key> --body <file>`  
+2.  Get Object  
+    `aws s3api get-object --bucket <your_bucket> --key <key> <save_to_file>`  
+3.  Delete Object  
+    `aws s3api delete-object --bucket <your_bucket> --key <key>`  
+4.  Head Object  
+    `aws s3api head-object --bucket <your_bucket> --key <key>`  
+5.  Put Object ACL  
+    `aws s3api put-object-acl --bucket <your_bucket> --key <key> --grant-* emailaddress=<email>`  
+6.  Get Object ACL  
+    `aws s3api get-object-acl --bucket <your_bucket> --key <key>`  
+7.  Delete multiple objects  
+    `aws s3api delete-objects --bucket <your_bucket> --delete <del_struct_json_file>`  
+8.  Put Object tagging  
+    `aws s3api put-object-tagging --bucket <your_bucket> --key <key> --tagging <tagging_json_struct>`  
+9.  Get Object tagging  
+    `aws s3api get-object-tagging --bucket <your_bucket> --key <key>`  
+10. Delete Object tagging  
+    `aws s3api delete-object-tagging --bucket <your_bucket> --key <key>`  
+
+
+# S3 IAM api (using client tool - s3iamcli)
+
+## Supported operations/IAM apis
+1.  Create an account  
+    `s3iamcli CreateAccount -n <Account Name> -e <Email Id>`  
+2.  Delete an account  
+    `s3iamcli DeleteAccount -n <Account Name>`  
+2.  List accounts  
+    `s3iamcli ListAccounts`  
+3.  Reset the access key/secret key of an account  
+    `s3iamcli ResetAccountAccessKey -n <Account Name>`  
+4.  Create IAM User, with optional path  
+    `s3iamcli CreateUser -n <User Name> [-p path]`  
+5.  Change/update the name of existing IAM user  
+    `s3iamcli UpdateUser -n <Old User Name> --new_user <New User Name> [-p <New Path>]`  
+6.  Delete IAM user  
+    `s3iamcli DeleteUser -n <User Name>`  
+7.  Create access key for IAM user  
+    `s3iamcli CreateAccessKey -n <User Name>`  
+8.  List access keys/secret keys of IAM user  
+    `s3iamcli ListAccessKeys -n <User Name>`  
+9.  Delete access key of IAM user  
+    `s3iamcli DeleteAccesskey -k <Access Key Id> -n <User Name>`  
+9.  Update the status of access key of IAM user (Change status to active or inactive)  
+    `s3iamcli UpdateAccessKey -k <Access Key Id> -s <Active/Inactive> -n <User Name>`  
+10. List all IAM users of current account, matching the path prefix  
+    `s3iamcli ListUsers [-p <Path Prefix>]`  
+11. Change the password of the IAM user who is calling this operation  
+    `s3iamcli ChangePassword --old_password <Old User Password> --new_password <New User Password>`  
+12. Create temporary authenitcation redentials for account or for IAM user, whichever is requested  
```

Comment:
it can be just Create temporary authenitcation credentials for account or for IAM user.
and there is typo in credentials above

Reply:
even typo in 'authenitcation'. Will fix it.